cmd/createorupdate/deploy.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/resources"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/log"
+)
+
+func deploy(cs *api.OpenShiftManagedCluster, p api.Plugin, azuredeploy []byte) error {
+	var t map[string]interface{}
+	err := json.Unmarshal(azuredeploy, &t)
+	if err != nil {
+		return err
+	}
+
+	authorizer, err := auth.NewAuthorizerFromEnvironment()
+	if err != nil {
+		return err
+	}
+
+	dcli := resources.NewDeploymentsClient(cs.Properties.AzProfile.SubscriptionID)
```

Comment:
How arm behaves here if you pass updated arm template? just for my info?

Reply:
That is what our testing is going to uncover ...
I think we start by severely gating the allowed configuration changes and start opening the gate bit by bit as we prove they work.  I'll add another commit to this PR.